Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageView.swift

```diff
@@ -8,7 +8,6 @@ struct EditorPageView: View {
     
     var body: some View {
         GenericUIKitToSwiftUIView(viewController: model.viewController)
-            .navigationBarHidden(true)
```

Comment:
In template selection screen. This call was breaking swipe. SwiftUI tried close navigation bar on container.
We close the nav panel in the container.